src/cloudflare/internal/images-api.ts

```diff
@@ -303,6 +327,165 @@ function chainStreams<T>(streams: ReadableStream<T>[]): ReadableStream<T> {
   return outputStream;
 }
 
+function concatUint8Arrays(a: Uint8Array, b: Uint8Array): Uint8Array {
+  const result = new Uint8Array(a.length + b.length);
+  result.set(a, 0);
+  result.set(b, a.length);
+  return result;
```

Comment:
This is going to become quite expensive as the buffers get bigger. If nodejs_compat was enabled then we could use `Buffer.concat` to be far more efficient.